BaseFilter/sections/antiadblock.txt

```diff
@@ -3412,6 +3412,8 @@ topsporter.net#%#//scriptlet('abort-current-inline-script', '$', 'keepChecking')
 topsporter.net#%#//scriptlet('prevent-xhr', 'static.doubleclick.net/instream/ad_status.js')
 !+ PLATFORM(ios, ext_android_cb, ext_safari)
 ||static.doubleclick.net/instream/ad_status.js$xmlhttprequest,redirect=nooptext,domain=topsporter.net
+! https://github.com/AdguardTeam/AdguardFilters/issues/176633
+||static.doubleclick.net/instream/ad_status.js$xmlhttprequest,redirect=nooptext,domain=trancehost.com

```

Comment:
Scriptlet is better solution, than redirect - it helps when used DNS filtering. Please add to existing rule.

Reply:
so we should use both, right?

Reply:
@Alex-302

Reply:
Only `@@||pagead2.googlesyndication.com/pagead/js/adsbygoogle.js$domain=` and sriptlet.